provider/internal/connections/resource_affinity_connection.go

```diff
@@ -30,58 +30,60 @@ import (
 var _ resource.Resource = &AffinityConnectionResource{}
 var _ resource.ResourceWithImportState = &AffinityConnectionResource{}
 
-func (t *AffinityConnectionResource) Schema(ctx context.Context, req resource.SchemaRequest, resp *resource.SchemaResponse) {
-	resp.Schema = schema.Schema{
-		MarkdownDescription: ":meta:subcategory:Connections: Affinity Connection",
-		Attributes: map[string]schema.Attribute{
-			"organization": schema.StringAttribute{
-				MarkdownDescription: "Organization ID",
-				Optional:            true,
-				Computed:            true,
-			},
-			"name": schema.StringAttribute{
-				Required: true,
-			},
-			"configuration": schema.SingleNestedAttribute{
-				Attributes: map[string]schema.Attribute{
-					"api_key": schema.StringAttribute{
-						MarkdownDescription: `API Key`,
-						Required:            true,
-						Optional:            false,
-						Computed:            false,
-						Sensitive:           true,
-						PlanModifiers: []planmodifier.String{
-							stringplanmodifier.UseStateForUnknown(),
-						},
-					},
-					"user": schema.StringAttribute{
-						MarkdownDescription: ``,
-						Required:            false,
-						Optional:            true,
-						Computed:            true,
-						Sensitive:           false,
+var AffinitySchema = schema.Schema{
+	MarkdownDescription: ":meta:subcategory:Connections: Affinity Connection",
+	Attributes: map[string]schema.Attribute{
+		"organization": schema.StringAttribute{
+			MarkdownDescription: "Organization ID",
+			Optional:            true,
+			Computed:            true,
+		},
+		"name": schema.StringAttribute{
+			Required: true,
+		},
+		"configuration": schema.SingleNestedAttribute{
+			Attributes: map[string]schema.Attribute{
+				"api_key": schema.StringAttribute{
+					MarkdownDescription: `API Key`,
+					Required:            true,
+					Optional:            false,
+					Computed:            false,
```

Comment:
I forget, did we discuss making these values `Computed` so that the masked values returned by the API wouldn't cause problems with state? Maybe that's not necessary with the other changes? I'm thinking about the case where someone is driving Polytomic _only_ from Terraform, so they're specifying the sensitive value using a variable, etc.

Reply:
I think we did, I'll update that to mark them as computed and validate that.